package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "importer",
+  "version": "1.0.0",
+  "description": "",
+  "main": "openhimConfig.js",
+  "scripts": {
+    "start": "node configBase.js",
+    "build": "esbuild configBase.js --bundle --platform=node --target=node10.4 --outfile=config.js"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Perhaps update this to reflect the license in the repo

Reply:
Oh yes, nice one. It's done